server/db.js

```diff
@@ -215,6 +237,28 @@ exports.createBackend = function createBackend(config) {
         });
       });
     },
+
+    getOps(project, callback) {
```

Comment:
Rename to e.g. getAllProjectOps?

Reply:
Use Caitlin's getOps(collabid, filepath), file collabs and files for project, call in sequence, return a map (collabid, filepath => ops sorted by v), client side take from each list in map on each iteration.